tests/core/fs/storage/test_user_storage.py

```diff
@@ -100,10 +103,13 @@ async def test_vacuum(alice_user_storage):
 
 @pytest.mark.trio
 @customize_fixtures(real_data_storage=True)
-async def test_storage_file_tree(tmp_path, alice):
+async def test_storage_file_tree(tmp_path: Path, alice: LocalDevice):
     manifest_sqlite_db = tmp_path / alice.slug / "user_data-v1.sqlite"
 
+    assert (
+        not manifest_sqlite_db.exists()
+    ), "database should not exist before the creation of user storage"
     async with UserStorage.run(tmp_path, alice) as aus:
-        assert aus.manifest_storage.path == manifest_sqlite_db
-
-    assert manifest_sqlite_db.is_file()
+        _ = aus.device
+        assert manifest_sqlite_db.exists(), "database file should exists"
+        assert manifest_sqlite_db.is_file(), "database should be a file"
```

Comment:
No need to call `exists()` before `is_file()`: `is_file()` will raise a `FileNotFoundError` if the file doesn't exist

Reply:
done